src/storage/txn/txn_status_cache.rs

```diff
@@ -122,18 +147,35 @@ use crate::storage::metrics::*;
 
 const TXN_STATUS_CACHE_SLOTS: usize = 128;
 
-/// An cache item should be kept for at least this time.
+/// A cache item should be kept for at least this time.
 /// Actually this should be guaranteed only for committed transactions. See
 /// [this section](#
 /// for-filtering-out-unwanted-late-arrived-stale-prewrite-requests) for details
 /// about why this is needed.
 const CACHE_ITEMS_REQUIRED_KEEP_TIME: Duration = Duration::from_secs(30);
+const CACHE_ITEMS_REQUIRED_KEEP_TIME_FOR_LARGE_TXNS: Duration = Duration::from_secs(30);
```

Comment:
The number of large transactions should be much less than normal transactions, perhaps this constant is unncessary if its value is the same.
While the `capacity` could be smaller for the large transactions.

Reply:
I use the same capacity mainly because I wanted to make the configs as few as possible. It can be confusing if we have 2 separate capacity config items for txn_status_cache. Besides, capacity merely sets an upper limit and doesn't directly correlate with the amount of memory actually used.

The individual constant just implies that the keep-time can be different and changed when needed.